merkle-distributor/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "merkle-distributor",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "test": "ts-node distributor.ts"
+  },
+  "author": "Han Zhao",
```

Comment:
Could you also attribute `litentry` here?

Reply:
sure, added.